src/main/java/com/minecolonies/items/ItemScanTool.java

```diff
@@ -1,16 +1,29 @@
 package com.minecolonies.items;
 
+import com.minecolonies.MineColonies;
 import com.minecolonies.creativetab.ModCreativeTabs;
+import com.minecolonies.lib.Constants;
+import com.minecolonies.network.messages.SaveScanMessage;
 import com.minecolonies.util.BlockPosUtil;
 import com.minecolonies.util.LanguageHandler;
-import com.minecolonies.util.SchematicWrapper;
+import com.minecolonies.util.StructureWrapper;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.util.StructureWrapper'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)